service/src/main/java/uk/gov/hmcts/reform/fpl/utils/EmailNotificationHelper.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.hmcts.reform.fpl.utils;
+
+import org.apache.commons.lang3.StringUtils;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.Respondent;
+import uk.gov.hmcts.reform.fpl.model.RespondentParty;
+import uk.gov.hmcts.reform.fpl.model.common.Element;
+
+import java.util.Objects;
+import java.util.stream.Stream;
+
+import static java.util.stream.Collectors.joining;
+import static org.apache.commons.lang3.ObjectUtils.defaultIfNull;
+import static org.springframework.util.CollectionUtils.isEmpty;
+
+public class EmailNotificationHelper {
+
+    private EmailNotificationHelper() {
+    }
+
+    public static String buildSubjectLine(final CaseData caseData) {
+        final String lastName = getRespondents1Lastname(caseData);
+        final String familyManCaseNumber = defaultIfNull(caseData.getFamilyManCaseNumber(), "");
+
+        return Stream.of(lastName, familyManCaseNumber)
+            .filter(StringUtils::isNotBlank)
+            .collect(joining(", "));
+    }
+
+    private static String getRespondents1Lastname(final CaseData caseData) {
```

Comment:
LastName instead of Lastname

Reply:
Actually whilst we are here, let's drop the 1 in respondents. 

"getFirstRespondentLastName"??

Reply:
done